src/components/tables/RowActions/Shared/Progress.tsx

```diff
@@ -13,22 +14,32 @@ export enum ProgressStates {
 export interface SharedProgressProps {
     name: string;
     error: any | null;
+    logToken: string | null;
     renderError?: Function;
+    renderLogs?: Function;
     successMessageID: string;
     runningMessageID: string;
     state: ProgressStates;
 }
 
+const wrapperStyling = { ml: 3, width: '100%' };
+
 function SharedProgress({
     name,
     error,
+    logToken,
     renderError,
+    renderLogs,
     state,
     successMessageID,
     runningMessageID,
 }: SharedProgressProps) {
     return (
-        <Box>
+        <Box
+            sx={{
+                pr: 3,
+            }}
+        >
```

Comment:
Putting a padding wrapper around so that the accordion for the logs does not run right up against the side of the dialog container.